tests/integration/Core/System/StateMachine/StateMachineRegistryTest.php

```diff
@@ -158,6 +162,58 @@ public function testStateMachineRegistryUnnecessaryTransition(): void
         static::assertEquals(OrderDeliveryStates::STATE_PARTIALLY_RETURNED, $toPlace->getTechnicalName());
     }
 
+    public function testStateMachineTransitionStoresUserAndIntegrationId(): void
+    {
+        $ids = new IdsCollection();
+
+        /** @var EntityRepository $userRepo */
```

Comment:
please do not use var annotations. Especially in tests you can simply assert the type. also see https://github.com/shopware/shopware/blob/trunk/coding-guidelines/core/writing-code-for-static-analysis.md